assets/model_monitoring/components/tests/e2e/utils/constants.py

```diff
@@ -113,7 +113,7 @@
 DATA_ASSET_MODEL_INPUTS_JOIN_COLUMN_NAME = 'model_inputs_join_column'
 DATA_ASSET_MODEL_OUTPUTS_JOIN_COLUMN_NAME = 'model_outputs_join_column'
 # Groundedness target dataset as a MLTable.
-DATA_ASSET_GROUNDEDNESS_PREPROCESSED_TARGET_DATA = 'azureml:groundedness_preprocess_target_small:1'
+DATA_ASSET_GROUNDEDNESS_PREPROCESSED_TARGET_DATA = 'azureml:mltable_groundedness_preprocessed_target_small:1'
```

Comment:
you didn't add groundtruth column to test data, I suppose you didn't include the groundness metrics in e2e test? Should we add that column and include groundness metrics? To let the e2e cover more code path.